File: Modul2/Python/PRAK204-2310817210003-Muhammad_Azrianzan.py
r, tinggi = input("").split()

r = float(r)
tinggi = float(tinggi)

r2 = float(r * r)
volume = float(3.142857 * r2 * tinggi)
luas = float(2 * 3.142857 * r * (r + tinggi))
keliling = float(2 * 3.142857 * r)

print(f"Volume : {volume:.2f}")
print(f"Luas : {luas:.2f}")
print(f"Keliling : {keliling:.2f}")

File: Modul2/C/PRAK204-2310817210003-Muhammad_Azrianzan.c
#include <stdio.h>

int main() {
    float r;
    float tinggi;
    printf("");
    scanf("%f %f", &r ,&tinggi);

    float r2 = r * r;
    float volume = 3.142857 * r2 * tinggi;
    float luas = 2 * 3.142857 * r * (r + tinggi);
    float keliling = 2 * 3.142857 * r;

    printf("Volume : %.2f\n", volume);
    printf("Luas : %.2f\n", luas);
    printf("Keliling : %.2f\n", keliling);

    return 0;
}